src/generic/evas/meson.build

```diff
@@ -36,11 +36,15 @@ foreach loader : generic_loaders
   if get_option('evas-loaders-disabler').contains(loader) == false
     subdir(loader)
 
+    evas_dep = [eina, generic_deps]
+    if sys_windows
+      evas_dep += [evil_unposix]
```

Comment:
Is this still relevant after the unposix patch?

Reply:
After the unposix patch, it will not be relevant, but as it isn't at `native-windows` yet, we need it.
If you guys prefer, I could make the patch depending on #131, do you think it would be better @walac, @joaoantoniocardoso?

Reply:
Either that or you could create an issue to remove this piece of code once #131 lands. Both approaches are fine by me.